packages/logo/src/Logo.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * A logo for Jetpack.
+ *
+ * @package jetpack-logo
+ */
+
+namespace Jetpack\Assets;
+
+/**
+ * Create and render a Jetpack logo.
+ */
+class Logo {
+	/**
+	 * Absolute URL of the Jetpack logo.
+	 *
+	 * @var string
+	 */
+	private $url;
+
+	/**
+	 * Constructor.
+	 * You can optionally pass a URL to override the default one.
+	 *
+	 * @param string $url New URL of the Jetpack logo.
+	 */
+	public function __construct( $url = '' ) {
+		if ( ! $url ) {
+			$url = plugins_url( 'assets/logo.svg', __DIR__ );
```

Comment:
Can we assume that each plugin that uses this library will have the logo in this place?

Reply:
it is relative to the file folder when you pass the second argument.

Reply:
and we are shipping `assets/logo.svg` with the package. 🤝